pep-0425.txt

```diff
@@ -109,7 +109,8 @@ ABI Tag
 The ABI tag indicates which Python ABI is required by any included
 extension modules.  For implementation-specific ABIs, the implementation
 is abbreviated in the same way as the Python Tag, e.g. `cp33d` would be
-the CPython 3.3 ABI with debugging.
+the CPython 3.3 ABI with debugging (calculated by concatenating the Python
+tag with `sys.abiflags()`).
```

Comment:
*gasp* You mean I don't have to go digging in `sysconfig` for that? 

Also, this is just a string, so the parentheses shouldn't be there.

What does `sys.abiflags` get set to on Windows?

Reply:
On Windows, "AttributeError: module 'sys' has no attribute 'abiflags'"